src/testRunner/runner.ts

```diff
@@ -210,6 +220,26 @@ ${JSON.stringify(dupes, undefined, 2)}`);
                         case "docker":
                             runners.push(new DockerfileRunner());
                             break;
+                        case "tsbuild":
+                            runners.push(new DynamicUnittestsRunner(DynamicUnittests.TestType.Tsbuild));
+                            runUnitTests = true;

```

Comment:
Setting this true for now till we can convert all tests so that its easier to run all tsbuild tests in single pass whether they are in unittests or not